pkg/util/bool.go

```diff
@@ -0,0 +1,9 @@
+package util
```

Comment:
Nit: We are probably going to need more of these at some point, perhaps rename this file to `pointers` so that we can add other types here too